Shootingat Resolved Project/Assets/Scripts/Characters/Core/Stats/Stat.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using TMPro;
+using UnityEngine;
+
+namespace PabloLario.Characters.Core.Stats
+{
+
+    // Forcing generic type to be number is not available until C# 10
```

Comment:
I would loved to avoid `IntStat` and `FloatStat` classes using a generic parameter which could apply numerical operations